apis/sharing/v1alpha1/resourceoffer_types.go

```diff
@@ -75,6 +75,7 @@ type ResourceOfferStatus struct {
 // +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.phase`
 // +kubebuilder:printcolumn:name="Expiration",type=string,JSONPath=`.spec.timeToLive`
```

Comment:
Does this TTL still exist? If not it should be dropped.

Reply:
I don't know. Maybe @aleoli and @palexster could help here.

Reply:
In my opinion, we can drop it